simulator/docs/record-and-replay-cluster-events.md

```diff
@@ -0,0 +1,95 @@
+# [Beta] Record your real cluster's events and replay them in the simulator
```

Comment:
You have to put the link for this page at somewhere (README?) so that people can notice